modules/cloudtrail/alarm-actions.tf

```diff
@@ -0,0 +1,13 @@
+module "cloudwatch_alarm_actions" {
```

Comment:
no need to have alarm actions in modules,
alarm actions create topic and its subscribers and we suppose this will be created in account level and all modules including this one just will have topic name to send alert information to